api/envoy/config/trace/v3/tracer.proto

```diff
@@ -0,0 +1,29 @@
+syntax = "proto3";
+
+package envoy.config.trace.v3;
+
+import "google/protobuf/any.proto";
+
+import "udpa/annotations/status.proto";
+import "udpa/annotations/versioning.proto";
+import "validate/validate.proto";
+
+option java_package = "io.envoyproxy.envoy.config.trace.v3";
+option java_outer_classname = "TracerProto";
+option java_multiple_files = true;
+option (udpa.annotations.file_status).package_version_status = ACTIVE;
+
+// [#protodoc-title: Tracing]
+// Tracing :ref:`architecture overview <arch_overview_tracing>`.
+
+// The tracing configuration specifies settings for an tracer provider used by Envoy.
+message TracerProvider {
```

Comment:
Where this is used?

And the content of this message is exactly same as `TypedExtensionConfig` so we can just use that.

Reply:
It's designed to replace `envoy.config.trace.v3.Tracing.Http` in the future. There is no place to use it yet. But you are right, we can use `TypedExtensionConfig` directly.